### PR TITLE
NyanoCombat 2, Part 3: Physical Traits

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

Done in partnership with @OldDanceJacket 

This PR adds 9 additional physical traits to the game, 5 positive, and 4 negative. While this PR is intended to go with NyanoCombat 2, Part 1 and 2, I have made this PR function completely standalone. It does not require either of the two other PRs.

## Positive Traits
- **Will To Live**: Increases your Dead threshold by 10
- **Tenacity**: Increases your Crit threshold by 5
- **Vigor**: Increases your maximum Stamina by 10
- **High Adrenaline**: You gain up to 10% more damage with all melee attacks when injured. 
- **Masochism**: You ignore the first 10% of stamina damage penalties to melee attacks. If NyanoCombat 2 Part 1 isn't merged yet, this makes you deal up to 10% more melee damage when you receive stamina damage.
- **Martial Artist**: Your unarmed melee attacks have bonus range, and deal 50% more damage(for every species in the game, this means 7.5 instead of 5 damage). This trait is identical to one that the Boxer job receives for free, thus it cannot be taken by Boxers.

## Negative Traits
- **Will To Die**: Decreases your Dead threshold by 15
- **Glass Jaw**: Decreases your Crit Threshold by 10
- **Lethargy**: Decreases your maximum Stamina by 15
- **Adrenal Dysfunction**: Your melee attacks are penalized by up to 20% when injured. If NyanoCombat 2 Part 1 is merged, this cancels out the natural bonus everyone globally gets to melee when injured.
- **Low Pain Tolerance**: Your melee attacks are penalized by up to 15% when receiving stamina damage. If NyanoCombat 2 Part 1 is merged, this stacks with the natural penalties everyone globally gets to melee when taking stamina damage.

# TODO

- [ ] Let ODJ look over these for balance.

# Media

image

# Changelog

:cl: VMSolidus and Skubman
- add: 11 new Physical Traits have been added to the game! 6 positive traits, and 5 negative traits.

### CODE REVIEW

**Comment by DangerRevolution** [comment]
Given this is a chonking PR, probably worth to have 3-4 approvals before merge :)

**Review by DangerRevolution — APPROVED** [review]

**Comment by angelofallars** [comment]
The changelog text needs to be updated because it's 11 traits now ~~and I made 2 of them so Skubman should be credited too :trollface:~~

**Comment by DangerRevolution** [comment]
Fixed the merge conflict so this shit can get reviewed

**Review by OldDanceJacket — APPROVED** [review]

**Pull request merged**
